### PR TITLE
Improvements 743 rework client side

### PR DESCRIPTION
**Pull request by RomanGalochkin**
Here, I have temporary turned off: "HelpDropdown" component and changed "Dashboard" on JSX2. We need to debug Jsx3 anyway. Bugs are possible.

### CODE REVIEW

File: packages/client/src/lib/dashboardTable/Dashboard.re [hunk 1]
```diff
@@ -25,32 +27,31 @@ type state = {text: string};
...
+[@react.component]
+let make = (~channelId: string) => {
+  //  ...component,
+  //  reducer: (action, _state) =>
+  //    switch (action) {
+  //    | Update(text) => ReasonReact.Update({text: text})
+  //    },
+  //
+  //  initialState: () => {text: ""},
+  //  render: self => {
```
Comment (OAGr) [inline packages/client/src/lib/dashboardTable/Dashboard.re]: are these lines still needed?

File: packages/client/src/lib/reform/ReForm.re [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+  };
+
+module type Config = {
+  type state;
+  type fields;
+  /* (fieldName, getter, setter) */
+  let lens: list((fields, state => Value.t, (state, Value.t) => state));
+};
+
+module Create = (Config: Config) => {
```
Comment (OAGr) [inline packages/client/src/lib/reform/ReForm.re]: Did you copy this from the ReForm codebase? Do you think there could be a more elegant way, like just having the jsx3 part?

**Comment by OAGr** [comment]
Overall this looks good to me. Make sure to fix deployment though, this currently seems to break Heroku.

File: packages/client/src/lib/dashboardTable/Dashboard.re (hunk 1 above)
Comment (RomanGalochkin) [reply packages/client/src/lib/dashboardTable/Dashboard.re]: I think yes, I am going to remove it on the next time.

File: packages/client/src/lib/reform/ReForm.re (hunk 2 above)
Comment (RomanGalochkin) [reply packages/client/src/lib/reform/ReForm.re]: Yes, I have copied the lib and added Jsx3. But now I realized that may be we can just add dedicated Jsx3 "shells" for these Jsx2 components.

**Comment by RomanGalochkin** [comment]
Yes, sure.

File: packages/client/src/lib/reform/ReForm.re (hunk 2 above)
Comment (OAGr) [reply packages/client/src/lib/reform/ReForm.re]: Makes sense. The Jsx3 shells sound good to me.

**Comment by RomanGalochkin** [comment]
I think that it is all with this issue. I have a bit modified "reason-react" for a temporary reason (https://github.com/RomanGalochkin/reason-react.git#0.7.0.beta.0) later we would get this package back. 

- On the next issue to take time and fix "ReFormNext" forms (to add Jsx3 adapters).
- Recover "FC_HelpDropdown" component.
- Recover hooks in Dashboard components.
- To fix another small issues. I think they are.

**Comment by OAGr** [comment]
Cool. I made a js3x branch for these issues, and will merge into that. Maybe make new issues targeting that branch?

**Comment by RomanGalochkin** [comment]
Sure.

**Pull request merged**